pipeline/README.md

```diff
@@ -0,0 +1,28 @@
+# pipeline
+
+Directory 'pipeline' contains all the CI/CD pipelines of image service.
+Currently, there are 2 pipelines: 
+- [BuildBaseImage](http://147.178.202.18/job/image-service/job/BuildBaseImage/)
+- [PR Gate](http://147.178.202.18/job/image-service/job/PR_Gate/)
+
+Copyright 2017, DELLEMC, Inc.
+
+## Build Base Image
+
+*The entry point of BuildBaseImage: pipeline/BuildBaseImage/Jenkinsfile*
+
+The pipeline is responsible for building a base docker image which contains all the prerequisites for image-service.
+Pipelines which run test with docker, such as PR Gate, will build docker from the base image. 
+That will save the test time.
```

Comment:
I think the key point is not to save time, but increase the reliability in case unstable network , right ?